html/ui/html/modals/account.html

```diff
@@ -85,7 +85,7 @@ <h4 class="modal-title" data-i18n="account_details">Account Details</h4>
           <table class="table" id="account_balance_table" style="margin-bottom:0;table-layout:fixed;">
             <tbody>
               <tr>
-                <td style="font-weight: bold;width:180px;"><span data-i18n="account_id">Account Address</span>:</td>
+                <td style="font-weight: bold;width:180px;"><span data-i18n="account_id">Account ID</span>:</td>
```

Comment:
I think we should merge recent changes on your branch before this merge. We are missing some recent changes. This is one and I will point out some others.